tests/data/sampler/largestTriangle3Spec.js

```diff
@@ -0,0 +1,62 @@
+describe('fc.data.sampler.largestTriangle3', function() {
+
+    var data;
+    var dataGenerator;
+
+    describe('simple xy data', function() {
+
+        beforeEach(function() {
+
+            data = [
+                { x: 0, y: 6 }, // Bucket 1
```

Comment:
I hand-worked these out, but I've also ran it on the codebase above and have similar results. They're not identical, however, due to slight implementation details.

For example, the first and last buckets in the above module are always first and last, meaning the first/last few data points are always discarded. To counteract this, I don't discard these data points, but evenly partition between elements `1..n-1`. These are the only differences in the expected output in the algorithm, the data points in the middle are identical otherwise :)